pkg/pod/ttlDestroyHandler.go

```diff
@@ -56,7 +57,7 @@ func (t ttlDestroyHandler) Handle(pod v1.Pod) error {
 	}
 
 	if t.ttlBeforeNow(creationTimestamp, ttl) {
-		id, err := strconv.ParseUint(pod.Labels["im-id"], 10, 64)
+		id, err := strconv.ParseUint(pod.Labels["im-instance-id"], 10, 64)
 		if err != nil {
 			return err
 		}
```

Comment:
The [issue](https://dhis2.atlassian.net/browse/DEVOPS-400) mentions

> set a correlation id on the amqp message and log it on the consumer side in the im-manager. This helps us tie together what message caused a TTL deletion to fail.

The id would be generated in here, put into the amqp message correlation id and as a log attribute on this side and then the consumer. This way you could find all logs related to the deletion of a particular pod that went wrong.